geomstats/backend/pytorch/__init__.py

```diff
@@ -320,6 +320,8 @@ def trace(*args, **kwargs):
 def mod(*args, **kwargs):
     return torch.fmod(*args, **kwargs)
 
+def arctanh(x):
```

Comment:
wow, I didn't know pytorch doesn't have this: https://github.com/pytorch/pytorch/issues/10324.
thanks!